financial-connections/src/main/java/com/stripe/android/financialconnections/features/networkinglinkloginwarmup/NetworkingLinkLoginWarmupViewModel.kt

```diff
@@ -73,7 +76,14 @@ internal class NetworkingLinkLoginWarmupViewModel @Inject constructor(
     private fun onSkipClicked() {
         suspend {
             eventTracker.track(Click("click.skip_sign_in", PANE))
-            disableNetworking().also { goNext(it.nextPane) }
+            disableNetworking().also {
+                navigationManager.navigate(
+                    NavigateToRoute(
+                        command = it.nextPane.toNavigationCommand(),
+                        popCurrentFromBackStack = true
```

Comment:
this was the main reason for this refactor, and we're expecting to run into cases like this in the future.